ui/page/root/overview_page.go

```diff
@@ -704,17 +807,17 @@ func (pg *OverviewPage) marketTableHeader(gtx C) D {
 			Alignment: layout.Middle,
 		}.Layout(gtx,
 			layout.Flexed(.8, func(gtx C) D {
-				return layout.E.Layout(gtx, pg.assetTableLabel("Price", col))
+				return layout.E.Layout(gtx, pg.assetTableLabel(values.String(values.StrPrice), col))
 			}),
 			layout.Flexed(.2, func(gtx C) D {
-				return layout.E.Layout(gtx, pg.assetTableLabel("ID Change", col))
+				return layout.E.Layout(gtx, pg.assetTableLabel(values.String(values.StrChange), col))
```

Comment:
This is actually meant to represent 1 day change i.e `1D Change`

Reply:
Should we use 24H Change instead? 1D might not be clear for most users, I think.

Reply:
Yes, this 24H is better